examples/common/imgui_ui/windows/light.cpp

```diff
@@ -65,9 +65,18 @@ ImVec4 HueSaturationToColor(float hueDegrees, float saturationPercent)
 
 void Light::UpdateState()
 {
+    if (mTargetLightIsOn.HasValue()) {
+        OnOff::Attributes::OnOff::Set(mEndpointId, mTargetLightIsOn.Value());
+        mTargetLightIsOn.ClearValue();
+    }
     OnOff::Attributes::OnOff::Get(mEndpointId, &mLightIsOn);
 
     // Level Control
+    if (mTargetLevel.HasValue()) {
+        LevelControl::Attributes::CurrentLevel::Set(mEndpointId, mTargetLevel.Value());
```

Comment:
Again, how does this interact with in-flight level transitions?  How does it interact with the spec-required interactions between level and on/off state?

Please make sure @jmartinez-silabs reviews this.